.github/workflows/release.yml

```diff
@@ -29,7 +29,10 @@ jobs:
         shell: bash
         run: ./scripts/parse_ref.sh ${{ github.ref }} >> ${GITHUB_OUTPUT}
 
-  build:
+  build-and-sign:
+    permissions:
+      # cosign uses the GitHub OIDC token
+      id-token: write
```

Comment:
What account is this set up for?

Reply:
The `id-token: write` permission enables the gh workflow to request an OIDC token generated from GitHub OIDC provider, which will then be used by `cosign` to authenticate with Sigstore for signing. `cosign` uses Fulcio, upon receiving the OIDC token, to validate the token. This validation includes checking the token's integrity, its issuer (GitHub in this case), and the claims within the token (like the repository and job details). 

After the validation step, Fulcio trusts that the signing request is legitimately coming from the specified GitHub Actions workflow.

So to answer your question, it does not set up an account for anyone. `cosign` backend verfies the emphameral token with GitHub OIDC provider.

Reply:
the verification doc might help clarify it for me

Reply:
```
cosign verify-blob --signature containerd-shim-wasmtime-v1.sig --certificate containerd-shim-wasmtime-v1-crt.pem  --certificate-identity https://github.com/Mossaka/runwasi/.github/workflows/release.yml@refs/tags/containerd-shim-wasmtime/v0.3.1 --certificate-oidc-issuer https://token.actions.githubusercontent.com  containerd-shim-wasmtime-v1
```

This is the command I used to verify the release binary on a forked repo when I was testing this PR.

Reply:
As the command shows, the identity is a reference to that workflow run which requests the token from Github OIDC. 
`--certificate-identity https://github.com/Mossaka/runwasi/.github/workflows/release.yml@refs/tags/containerd-shim-wasmtime/v0.3.1`

Reply:
Added instructions to verify blobs on RELEASE.MD

Reply:
> As the command shows, the identity is a reference to that workflow run which requests the token from Github OIDC. `--certificate-identity https://github.com/Mossaka/runwasi/.github/workflows/release.yml@refs/tags/containerd-shim-wasmtime/v0.3.1`

got it thanks!  This does seem a little strange to me to be the cert identity 🤷